From 8734cc1aa6e7730db0965cb1fa825b5acf7570a3 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 11:59:38 +0800
Subject: [PATCH] tweak bootloader fs type

Use ext2 as default fs of bootloader to workaround grub-probe
failure while e2fsprogs upgraded to 1.43

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/modules/storage/bootloader/grub2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/modules/storage/bootloader/grub2.py b/pyanaconda/modules/storage/bootloader/grub2.py
index 47a3a6d55..46c03af68 100644
--- a/pyanaconda/modules/storage/bootloader/grub2.py
+++ b/pyanaconda/modules/storage/bootloader/grub2.py
@@ -186,7 +186,7 @@ class GRUB2(BootLoader):
         if productName.startswith("Red Hat "): # pylint: disable=no-member
             return ["xfs", "ext4", "ext3", "ext2"]
         else:
-            return ["ext4", "ext3", "ext2", "btrfs", "xfs"]
+            return ["ext2", "ext3", "ext4", "btrfs", "xfs"]
 
     #
     # grub-related conveniences
-- 
2.7.4

